Documentation/policy/language.rst

```diff
@@ -1119,21 +1119,21 @@ If multiple allow and deny policies are applied to the same pod, the following
 table represents the expected enforcement for that Pod:
 
 +--------------------------------------------------------------------------------------------+
-| **Ingress Policies Deployed to Server Pod (Yes / No)**                                     |
+| **Set of Ingress Policies Deployed to Server Pod**                                         |
 +---------------------+-----------------------+---------+---------+--------+--------+--------+
-|                     | Layer 7 (HTTP)        | Yes     | Yes     | Yes    | Yes    | No     |
+|                     | Layer 7 (HTTP)        | X       | X       | X      | X      |        |
```

Comment:
Maybe let's use `✓` instead?

Reply:
Yep, looks good enough to me:
![image](https://user-images.githubusercontent.com/1764210/115857244-bdd0e680-a42d-11eb-80a8-5c8acf7a7d87.png)